history-of-quakers/modernized/22-chapter-20.adoc

```diff
@@ -1148,8 +1261,18 @@ which the apostles and primitive Christians used, saying,
 "`The weapons of our warfare are not carnal, but mighty through God,`" that is,
 spiritual, 2 Cor. 10:4.
 
-G+++.+++ FOX
+[.signed-section-signature]
+G+++.+++ Fox
 
+--
+
+[.small-break]
+'''
```

Comment:
hmmm... i think let's kill the small break here. on the pdf at least it (combined with the two back to back embeds) leaves a pretty large gap. might not look the same in the preview in the editor though.